numpy/core/src/multiarray/_multiarray_tests.c.src

```diff
@@ -2363,6 +2363,35 @@ run_intp_converter(PyObject* NPY_UNUSED(self), PyObject *args)
     return tup;
 }
 
+PyObject *
+test_scalar_intp_converter(PyObject *dummy, PyObject *obj)
+{
+    PyArray_Dims dummy_dims;
+    int output = PyArray_IntpConverter(obj, &dummy_dims);
+    if (output != NPY_SUCCEED) {
+        return NULL;
+    } else {
+        return PyLong_FromLong(dummy_dims.ptr[0]);
+    }
+}
+
+PyObject *
+test_scalar_intp_from_sequence(PyObject *dummy, PyObject *obj)
+{
+    npy_intp vals[1];
+    if (vals == NULL) {
+        PyErr_NoMemory();
+        return NPY_FAIL;
+    }
```

Comment:
```suggestion
```
This can't fail.  If you write something like `npy_intp vals[1]` C places it on the stack and not the heap.  That means there is no call to `malloc` that could fail.  (The "downside" is that the stack has a limited size, so you can't put can only put smallish amounts of data on it.)